src/main/resources/bidder-config/emxdigital.yaml

```diff
@@ -1,8 +1,11 @@
 adapters:
   emxdigital:
     endpoint: https://hb.emxdgt.com
+    aliases:
+      cadentaperturemx:
```

Comment:
Also from go code, looks like thay expect 'cadent_aperture_mx'

Reply:
I thought we could go with our own naming, couldn't we? We still have a helper endpoint to find all the bidders and aliases